inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java

```diff
@@ -40,7 +40,7 @@ public class Constants {
 
     public static final String DRAIN = "flink.drain";
 
-    public static final String METRICS_AUDIT_PROXY_HOSTS = "metrics.audit.proxy.hosts";
+    public static final String METRICS_AUDIT_PROXY_HOSTS_KEY = "metrics.audit.proxy.hosts";
```

Comment:
Can we remove this constant, and using the same constant from the `inlong-common` module?
I verified that the `manager-plugins` module has already dependent the `inlong-common` module.

Reply:
Hi @healchow 
Thanks for the suggestions. Have taken care of these in new commit. Please check

Reply:
LGTM